modules/login/php/requestaccount.class.inc

```diff
@@ -170,9 +170,11 @@ class RequestAccount extends \NDB_Form
                 )
             );
 
-            if ($_REQUEST['examiner'] == 'on') {
+            if (isset($_REQUEST['examiner']) && $_REQUEST['examiner'] == 'on') {
```

Comment:
```suggestion
            if (($_REQUEST['examiner'] ?? 'off') === 'on') {
```

Probably cleaner to use null coalesce to coalesce onto a default value. (same with below, but that one's spread across multiple lines so I can't recommend changes..)

Reply:
I dont like it with null coalesce inside the if

if you want I can define them in a variable above

Reply:
sure, use a variable. Either is cleaner than isset, I think.